### DISCUSSION TITLE
Portable mode

### DISCUSSION THREAD

**Discussion by itagagaki** [itagagaki/charu3-SE#43] [Ideas]
I changed the location of the preferences file (and so on) to %UserProfile% on Apr 21.
However, I will consider allowing users to choose portable mode.

**Comment by Lichtenshtein** [comment]
> I will consider allowing users to choose portable mode.

Yes, that would be great.

**Comment by itagagaki** [comment]
I have tried to activate the portable mode.
Place a plain file named `FORCE_INTO_PORTABLE_MODE` in the folder where you want to run Charu3.exe. When you then run Charu3.exe, it will be in portable mode and the settings and log files will be read and written to the folder with the Charu3.exe instead of %UserProfile% folder.
Remove or rename the `FORCE_INTO_PORTABLE_MODE` file and restart Charu3.exe to return to normal APPDATA mode.

Could you test it with what I have attached here?
I hope it works.

[Charu3_branch53.zip](https://github.com/itagagaki/charu3-SE/files/12498095/Charu3_branch53.zip)

**Comment by itagagaki** [comment]
This feature has been merged into the master branch for the next 3.9.0 release. Thank you!
